alert/tests.py

```diff
@@ -399,6 +402,95 @@ def test_wrong_user(self, mock_alert):
         self.assertEqual(0, CourseUpdate.objects.count())
 
 
+@override_settings(SWITCHBOARD_TEST_APP='pca')
+class APIRegisterTestCase(TestCase):
+    def setUp(self):
+        set_semester()
+        self.client = Client()
+        self.course, self.section = get_course_and_section('CIS-120-001', TEST_SEMESTER)
+        self.permission = APIPrivelege.objects.get_or_create(code=PCA_REGISTRATION)[0]
```

Comment:
I know this is a test case and it doesn't really matter, but I think you should just use `create` not `get_or_create` because the privilege can't exist unless you define it in the test case.